### PR TITLE
{lib}[intel/2015a,foss/2015a] XML related libaries + deps (REVIEW)

### PR DESCRIPTION
**Pull request by gjbex**
* libxml2
* libxslt
* Xerces C++
* XQilla
* cURL (foss)

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/00fc15a3e798e0245b12 for a full test report.

**Comment by wpoely86** [comment]
ok to test

**Comment by wpoely86** [comment]
@boegel if Jenkins give green light, ready to merge

**Comment by boegel** [comment]
missing dep `libxml2/2.9.2-foss-2015a`, @gjbex: please include a reference to which PR is providing this, or include it in here

I'll do what I can to tackle/merge your PRs ASAP, and retrigger Jenkins as PRs that provide dependencies for other get in

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+homepage = 'http://xerces.apache.org/xerces-c/'
+description = """Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = ['xerces-c-%(version)s.tar.gz']
+source_urls = [
+    'http://apache.belnet.be/xerces/c/3/sources/'
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.1-intel-2015a.eb]: one single empty line suffices

File: easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.1-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+homepage = 'http://xerces.apache.org/xerces-c/'
+description = """Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = ['xerces-c-%(version)s.tar.gz']
+source_urls = [
+    'http://apache.belnet.be/xerces/c/3/sources/'
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.1-foss-2015a.eb]: one single empty line suffices

**Comment by gjbex** [comment]
Fixed all.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b3365fa666d312154929 for a full test report.

**Comment by boegel** [comment]
Looking great, going in, thanks @gjbex!

**Pull request merged**